src/Sarif.FunctionalTests/SpecExamples/RuleMetadata.sarif

```diff
@@ -3,15 +3,25 @@
   "runs": [
     {
       "tool": {
-        "name": "CodeScanner",
+        "name": "BinaryAnalyzer",
         "semanticVersion": "2.1.0"
       },
       "rules": {
-        "C2001": {
-          "id": "C2001",
-          "fullDescription": "A variable was used without being initialized. This can result in runtime errors such as null reference exceptions"
+        "BA2006": {
+          "id": "BA2006",
+          "name": "BuildWithSecureTools",
+          "shortDescription": "Application code should be compiled with the most up-to-date tool sets ...",
+          "fullDescription": "Application code should be compiled with the most up-to-date tool sets ...",
```

Comment:
I got this text straight from BinSkim. And even if "toolset" is acceptable, surely "tool set" is also acceptable.

---
In reply to: [72288198](https://github.com/Microsoft/sarif-sdk/pull/464#discussion_r72288198) [](ancestors = 72288198)